### PR TITLE
Extend `linkify-user-location` to search results

### PR DESCRIPTION
**Pull request by kidonng**
Extracting stuff from #5113

Also match the style with GitHub on profile page and in hovercards.

## Test URLs

- https://github.com/search?q=sindre&type=users
- https://github.com/somebee

## Screenshot

https://user-images.githubusercontent.com/44045911/144449131-f6883dba-3411-42d7-87ee-5536ca9e7fc8.mov

### CODE REVIEW

File: source/features/linkify-user-location.tsx [hunk 1]
```diff
@@ -32,6 +37,20 @@ function init(): void {
...
+			linkifyLocation(possibleLocation, 'Link--muted');
+		}
+	}
+}
+
 void features.add(import.meta.url, {
 	init,
+}, {
+	include: [
+		() => pageDetect.isGlobalSearchResults() && new URLSearchParams(location.search).get('type') === 'users',
```
Comment (fregante) [inline source/features/linkify-user-location.tsx:53]: You can split this and place them in `asLongAs`

**Review by fregante — APPROVED** [review]

File: source/features/linkify-user-location.tsx [hunk 2]
```diff
@@ -32,6 +37,23 @@ function init(): void {
 	}
 }
 
+function searchInit(): void {
+	for (const possibleLocation of select.all('#user_search_results .text-small .mr-3:first-child')) {
```
Comment (fregante) [inline source/features/linkify-user-location.tsx:41]: Looking forward to this selector breaking. YOLO

**Review by fregante — APPROVED** [review]
Captains stay on the boat

**Comment by kidonng** [comment]
I noticed some other pages this would apply: followers/following pages.

File: source/features/linkify-user-location.tsx (hunk 2 above)
Comment (kidonng) [reply source/features/linkify-user-location.tsx:41]: First two parts should be reliable enough, I will see how to choose a better last part.

**Comment by kidonng** [comment]
Closing until I got the time to finish it.

**Pull request closed without merging**